terraform/api_gateway.tf.json.template.py

```diff
@@ -261,91 +328,126 @@ def check_waf_rules(rules: JSONs) -> JSONs:
                                                 'response_header': [
                                                     {
                                                         'name': 'Retry-After',
-                                                        'value': str(file_download_limit.retry_after),
+                                                        'value': str(rate_limit.retry_after)
                                                     }
                                                 ]
                                             }
                                         }
                                     },
                                     'visibility_config': {
-                                        'metric_name': 'FileDownloadRateLimit',
+                                        'metric_name': rate_limit.name,
                                         'sampled_requests_enabled': True,
                                         'cloudwatch_metrics_enabled': True
                                     }
-                                }
-                            ]),
+                                } for rate_limit in [config.waf_rate_limit_files]
```

Comment:
F.

Also, as mentioned in previous PRs, please don't create these redundant comprehensions.